tasks/configure.yml

```diff
@@ -9,6 +9,16 @@
     mode: 0700
   register: pgdata_dir_exist
 
+- name: PostgreSQL | Get locales
+  shell: locale -a
+  register: locales
+  changed_when: false
+
+- name: PostgreSQL | Ensure the locale is generated
+  sudo: yes
+  shell: locale-gen {{postgresql_locale.split('.').0}}
+  when: "'{{postgresql_locale.split('.').0 }}' not in locales.stdout"
+
```

Comment:
The [locale_gen](http://docs.ansible.com/locale_gen_module.html) module was added in Ansible 1.6, is there any interest to keep the compatibility with Ansible < 1.6?